src/standalone/Snow/Snow.jl

```diff
@@ -178,12 +179,12 @@ snow cover fraction in GCMs." Advances in Atmospheric Sciences
 
 where z̃ = snow depth per ground area / 0.106 m, and β_scf
 is computed using a resolution dependent formula:
-β_scf = max(β0 - γ(horz_degree_res - 1.5), β_min), where horz_degree_res is the
-horizontal resolution of the simulation, in degrees, and β0, β_min and γ
-are unitless. It is correct to think of β0, β_min, γ, and z0 as the free
+β\\_scf = max(β0 - γ(horz\\_degree\\_res - 1.5), β\\_min), where horz\\_degree\\_res is the
+horizontal resolution of the simulation, in degrees, and β0, β\\_min and γ
+are unitless. It is correct to think of β0, β\\_min, γ, and z0 as the free
 parameters, while horz_degree_res is provided and β_scf is determined.
 
-β0, β_min, γ, and β_scf must be > 0.
+β0, β_min, γ, and β\\_scf must be > 0.
```

Comment:
This docstring is still looking a little off, but I think it just needs `β\\_scf` in line 180 and `β\\_min` in line 187